frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -436,40 +441,32 @@ private void prepareStandardView(RenderRequest request) throws IOException {
 
         List<Project> projectList;
 
-        try {
-            final User user = UserCacheHolder.getUserFromRequest(request);
-            ProjectService.Iface projectClient = thriftClients.makeProjectClient();
-
-            String groupFilterValue = request.getParameter(Project._Fields.BUSINESS_UNIT.toString());
-            if (null == groupFilterValue) {
-                addStickyProjectGroupToFilters(request, user, filterMap);
-            } else {
-                ProjectPortletUtils.saveStickyProjectGroup(request, user, groupFilterValue);
-            }
+        final User user = UserCacheHolder.getUserFromRequest(request);
+        ProjectService.Iface projectClient = thriftClients.makeProjectClient();
 
+        String groupFilterValue = request.getParameter(Project._Fields.BUSINESS_UNIT.toString());
+        if (null == groupFilterValue) {
+            addStickyProjectGroupToFilters(request, user, filterMap);
+        } else {
+            ProjectPortletUtils.saveStickyProjectGroup(request, user, groupFilterValue);
+        }
+        try {
             if (isNullOrEmpty(searchtext) && filterMap.isEmpty()) {
                 projectList = projectClient.getAccessibleProjectsSummary(user);
             } else {
                 projectList = projectClient.refineSearch(searchtext, filterMap, user);
             }
-            for(Project project:projectList){
+            for (Project project : projectList) {
```

Comment:
Tell that intelliJ! (Or: you shall not press Ctrl+Alt+L any more without marking the appropriate area.)